src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
Wait how does "A dataset can reference objects in a another dataset.  It's not
a wall." fit with unique ownership of datasets?

Also, in that case, dataset should be called DataSubmission, since that is indeed unambiguous. 

Sorry, I have decided to wage a war against fuzzy terminology this morning.